examples/apps/generic_client/generic_client.py

```diff
@@ -116,14 +115,203 @@ def ParseCommandLine(args):
         json.dumps(config)
     except pconfig.ConfigurationException as e:
         logger.error(str(e))
+        config = None
+
+    return config
+
+
+def _retrieve_uri_from_registry_list(config):
+    # Retrieve Http JSON RPC listener uri from registry
+    logger.info("\n Retrieve Http JSON RPC listener uri from registry \n")
+    # Get block chain type
+    blockchain_type = config['blockchain']['type']
+    if blockchain_type == "Ethereum":
+            worker_registry_list = EthereumWorkerRegistryListClientImpl(
+                config)
+    else:
+        worker_registry_list = None
+        logger.error("\n Worker registry list is currently supported only for "
+                     "ethereum block chain \n")
+        return None
+
+    # Lookup returns tuple, first element is number of registries and
+    # second is element is lookup tag and
+    # third is list of organization ids.
+    registry_count, lookup_tag, registry_list = \
+        worker_registry_list.registry_lookup()
+    logger.info("\n Registry lookup response: registry count: {} "
+                "lookup tag: {} registry list: {}\n".format(
+                    registry_count, lookup_tag, registry_list))
+    if (registry_count == 0):
+        logger.error("No registries found")
+        return None
+    # Retrieve the fist registry details.
+    registry_retrieve_result = worker_registry_list.registry_retrieve(
+        registry_list[0])
+    logger.info("\n Registry retrieve response: {}\n".format(
+        registry_retrieve_result
+    ))
+
+    return registry_retrieve_result[0]
+
+
+def _lookup_first_worker(worker_registry, jrpc_req_id):
```

Comment:
Is the method name _lookup_first_worker make sense if we go for multiple workers in future. Change it to lookup_worker and mention comment saying currently one is supported and getting first worker.

Reply:
This function is to choose first worker as default worker if worker_id is not provided as an argument to generic client. I don't except this default behavior to change any time soon.